apps/wing/src/commands/pack.ts

```diff
@@ -120,6 +120,13 @@ export async function pack(options: PackageOptions = {}): Promise<string> {
       }
     }
 
+    // check package.json has `dependencies`
+    if (pkgJson.dependencies) {
+      throw new Error(
+        `Wing libraries should use "peerDependencies" instead of "dependencies" in package.json.`
+      );
```

Comment:
We often write error messages as "problem, solution". So maybe

```suggestion
      throw new Error(
        `Cannot create package with "dependencies" in package.json. Use "peerDependencies" instead.`
      );
```